src/main/util/file_util.py

```diff
@@ -303,3 +303,7 @@ def copy_file(src: str, dst: str) -> None:
     if is_file(src):
         create_directory(get_parent_folder(dst))
         copyfile(src, dst)
+
+
+def move_directory(src: str, dst: str) -> None:
+    shutil.move(src, dst)
```

Comment:
Why do you need this method?

Reply:
You are right, I forgot to delete this function. Thanks!